### ISSUE TITLE
nextcloud.apache keep crashes

### ISSUE THREAD

**Issue by miamilabs** [mail-in-a-box/mailinabox#1986]
Hey Community,

i did discover in my logs that "nextcloud.apache" keep trying to start and stop. 
Currently the port 80 is blocked by nginx.

> COMMAND  PID     USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
nginx   2614     root    8u  IPv4  36970      0t0  TCP *:http (LISTEN)
nginx   2614     root    9u  IPv6  36971      0t0  TCP *:http (LISTEN)
nginx   2618 www-data    8u  IPv4  36970      0t0  TCP *:http (LISTEN)
nginx   2618 www-data    9u  IPv6  36971      0t0  TCP *:http (LISTEN)
nginx   2619 www-data    8u  IPv4  36970      0t0  TCP *:http (LISTEN)
nginx   2619 www-data    9u  IPv6  36971      0t0  TCP *:http (LISTEN)

Did i make some mistake while setup?
Used same steps on another server and i am not facing this issue over there.

System : 
Ubuntu 18.04

Mail in a box :
0.53

Thank you all.

Attaching the server logs : 

<img width="833" alt="Screenshot 2021-05-27 at 10 00 46" src="https://user-images.githubusercontent.com/8977700/119789150-184cdf00-bed3-11eb-9261-4ad245826776.png">
 

> May 27 11:00:31  nextcloud.apache[61371]: Making sure nextcloud is setup...
May 27 11:00:31  nextcloud.apache[61371]: System config value redis => host set to string /tmp/sockets/redis.sock
May 27 11:00:32  nextcloud.apache[61371]: System config value redis => port set to integer 0
May 27 11:00:32  nextcloud.apache[61371]: System config value memcache.locking set to string \OC\Memcache\Redis
May 27 11:00:32  nextcloud.apache[61371]: System config value memcache.local set to string \OC\Memcache\Redis
May 27 11:00:32  nextcloud.apache[61371]: No such app enabled: updatenotification
May 27 11:00:33  nextcloud.apache[61371]: Making sure nextcloud is fully upgraded...
May 27 11:00:33  nextcloud.apache[61371]: Nextcloud is already latest version
May 27 11:00:33  nextcloud.apache[61371]: All set! Running httpd...
May 27 11:00:33  nextcloud.apache[61371]: No certificates are active: using HTTP only
May 27 11:00:33  nextcloud.apache[61371]: (98)Address already in use: AH00072: make_sock: could not bind to address [::]:80
May 27 11:00:33  nextcloud.apache[61371]: (98)Address already in use: AH00072: make_sock: could not bind to address 0.0.0.0:80
May 27 11:00:33  nextcloud.apache[61371]: no listening sockets available, shutting down
May 27 11:00:33  nextcloud.apache[61371]: AH00015: Unable to open logs
May 27 11:00:33  systemd[1]: snap.nextcloud.apache.service: Main process exited, code=exited, status=1/FAILURE
May 27 11:00:33  systemd[1]: snap.nextcloud.apache.service: Failed with result 'exit-code'.
May 27 11:00:33  systemd[1]: snap.nextcloud.apache.service: Service hold-off time over, scheduling restart.
May 27 11:00:33  systemd[1]: snap.nextcloud.apache.service: Scheduled restart job, restart counter is at 86.
May 27 11:00:33  systemd[1]: Stopped Service for snap application nextcloud.apache.
May 27 11:00:33  systemd[1]: Started Service for snap application nextcloud.apache.

**Comment by jvolkenant** [comment]
My guess is you did not start with a fresh install.  Something installed nextcloud.apache snap application

**Comment by miamilabs** [comment]
> My guess is you did not start with a fresh install. Something installed nextcloud.apache snap application

This VM was brand new and it runs only mail in a box. Only UNIX packages get updated and nothing else.
As i understand nextcloud.apache snap is not part of the installation package, i will deactivate it.

Thank you

**Comment by miamilabs** [comment]
> My guess is you did not start with a fresh install. Something installed nextcloud.apache snap application

This VM was brand new and it runs only mail in a box. Only UNIX packages get updated and nothing else.
As i understand nextcloud.apache snap is not part of the installation package, i will deactivate it.

Thank you
